test/unit/smartdown_content/smartdown_scenarios_test.rb

```diff
@@ -56,4 +57,15 @@ def outcome_is_reached(flow, answers, outcome)
                    "In flow #{flow.name} with answers #{answers}, outcome #{state.current_node.name} was reached and not #{outcome}"
     end
   end
+
+  def markers_are_included(flow, answers, markers)
```

Comment:
the method name denotes that it's checking for inclusion, but it's actually checking if the elements are equal. I'd either change the name of the method to `markers_match`, `assert_all_markers`, `markers_equal` etc. or modify the method to check for inclusion. Depends on what you want I guess :)